src/components/Reactions/ReportActionItemEmojiReactions.js

```diff
@@ -28,12 +28,16 @@ const propTypes = {
      */
     toggleReaction: PropTypes.func.isRequired,
 
+    /** A bool indicating if the user can add or remove reactions from current action */
```

Comment:
again, might be good to provide some more context on why adding/removing reactions might be disabled for a comment